### PR TITLE
Gives the Corporate Liaison a map-specific lore-brief about the map.

### PR DESCRIPTION
**Pull request by Detective-Google**
# About the pull request

This PR adds a new variable to map configs, called liaison_briefing, which is inputted into a piece of paper that spawns on the liaison's fax machine roundstart.

The idea is to insert a bit of lore about the colony the marines are going to, as well as to give the liaison some roleplay primers to start the round with.

If the map does not have a "liaison_briefing" set, it will default to a placeholder message explaining that there isn't a colony briefing available for that colony (todo; pretty the placeholder up)

For now, I've only written the trijent one, as it had the most concrete existing lore I could pull from to write a document about, some of the other maps (like LV, especially) don't have very well established lore, and will need working out with the lore guys on what best to include. If you want to write a briefing for your/your favorite map, feel free, I recommend preparing it in paperwork sim; [paperwork sim](http://ps.ss13.net/#instructions), and I'll be more than happy to convert it to HTML and add it to this PR or PR it for you :)
# Explain why it's good for the game

It always seemed weird to me that a lot of the time the CL has very little clue about this colony that they supposedly care about and don't want to get bombed to the stone age, out of everyone on the Almayer, the Liaison's ties to the company mean he should likely be the best informed about the colony. To this end, I've created a framework to brief the liaison about the map and it's interesting quirks, as well as giving them some fluff about why it may have gone dark(which conveniently never mentions xenomorphs.) My hope is that this'll aid the liaison in seeming knowledgable about the colony, as well as provide an avenue to inject some of the lore directly into the game, instead of being on a somewhat obscure subpage of a subpage of the wiki.

# Testing Photographs and Procedure

Spawn in on map with a liaison_briefing set, verify the briefing spawns, and is properly filled out, spawn on a map without a liaison_briefing set, verify the briefing spawns, and is populated with the placeholder explaining the lack of a briefing.

<details>
<summary>Screenshots & Videos</summary>
Ingame appearance of the trijent brief:

zhlFP0BGLC
paperwork sim view of the trijent brief:

image
placeholder briefing unavailable brief:

image

</details>

# Changelog
:cl:detectivegoogle
add: the CL now gets a briefing about the lore of the active map, assuming one has been written.
/:cl:

### CODE REVIEW

**Comment by Detective-Google** [comment]
image

updated message for maps without a briefing set

**Comment by Detective-Google** [comment]
image

a briefing for Kutjevo has been written, pulling info from the really nice Kutjevo lore doc

**Review by harryob — CHANGES_REQUESTED** [review]
this won't work on the prod server as `` is not using the cdn. on the production server, you get assets served to you on a funny url like http://cdn.cm-ss13.com/live/00/asset.00347bf0e1b4d6aa3adc389ddf49f4a6.png (see? random tacmap)

the best way would be to specify in the json a html file stored elsewhere in the repo, ie
`"liaison_briefing": "html/briefings/trijent.html"`

then you replace "wylogo.png" with %%WYLOGO%%

```html
<center>
    <img src="%%WYLOGO%%" /><br /><small><b>Weyland-Yutani Corporation</b><br />\"Building Better Worlds\"</small>
</center>
<br /><u>Special Services Division</u><br />Liaison Colony Briefing<br />

<!-- and so on... -->

```

in your `/obj/item/paper/liaison_brief/Initialize()` you would do

```dm
/obj/item/paper/liaison_brief/Initialize(mapload, ...)
	. = ..()
	if(SSmapping.configs[GROUND_MAP].liaison_briefing)
		var/datum/asset/asset = get_asset_datum(/datum/asset/simple/paper)

		info = file2text(SSmapping.configs[GROUND_MAP].liaison_briefing)
		info = replacetext(info, "%%WYLOGO%%", asset.get_url_mappings()["wylogo.png"])
	else
		// fallback code

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -913,3 +913,22 @@
 	. = ..()
 	info = "<div> <img style='align:middle' src='[SSassets.transport.get_asset_url("colonialspacegruntsEZ.png")]'>"
 	update_icon()
+
+/obj/item/paper/liaison_brief
+	name = "Liaison Colony Briefing"
+	desc = "A brief from the Company about the colony the ship is responding to."
+	var/placeholder = "maps/map_briefings/cl_brief_placeholder.html"
+	icon_state = "paper_wy_words"
```
Comment (harryob) [inline code/modules/paperwork/paper.dm:921]:
```suggestion
	icon_state = "paper_wy_words"
	
	var/placeholder = "maps/map_briefings/cl_brief_placeholder.html"
```

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -913,3 +913,22 @@
...
+	if(SSmapping.configs[GROUND_MAP].liaison_briefing)
+		var/datum/asset/asset = get_asset_datum(/datum/asset/simple/paper)
+
+		info = file2text(SSmapping.configs[GROUND_MAP].liaison_briefing)
+		info = replacetext(info, "%%WYLOGO%%", asset.get_url_mappings()["wylogo.png"])
+	else
+		var/datum/asset/asset = get_asset_datum(/datum/asset/simple/paper)
+
+		info = file2text(placeholder)
+		info = replacetext(info, "%%WYLOGO%%", asset.get_url_mappings()["wylogo.png"])
```
Comment (harryob) [inline code/modules/paperwork/paper.dm:934]:
```suggestion
		info = file2text(SSmapping.configs[GROUND_MAP].liaison_briefing)
	else
		info = file2text(placeholder)
		
	var/datum/asset/asset = get_asset_datum(/datum/asset/simple/paper)
	info = replacetext(info, "%%WYLOGO%%", asset.get_url_mappings()["wylogo.png"])
```

File: maps/map_briefings/cl_brief_trijent.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<center>
+	<img src="wylogo.png" /><br /><small
```
Comment (harryob) [inline maps/map_briefings/cl_brief_trijent.html:2]:
```suggestion
	<img src="%%WYLOGO%%" /><br /><small
```

File: maps/map_briefings/cl_brief_placeholder.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<center>
+	<img src="wylogo.png" /><small
```
Comment (harryob) [inline maps/map_briefings/cl_brief_placeholder.html:2]:
```suggestion
	<img src="%%WYLOGO%%" /><small
```

File: maps/map_briefings/cl_brief_kutjevo.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
+<center>
+	<img src="wylogo.png" /><small
```
Comment (harryob) [inline maps/map_briefings/cl_brief_kutjevo.html:2]:
```suggestion
	<img src="%%WYLOGO%%" /><small
```

**Review by harryob — CHANGES_REQUESTED** [review]
last bits

**Review by harryob — APPROVED** [review]

**Pull request merged**